models.py

```diff
@@ -20,7 +26,7 @@ class Story(Base):
     __tablename__ = 'stories'
     id = Column(Integer, primary_key=True)
     story = Column(String)
```

Comment:
This field is not a relationship, having story field in a user model while it's not a relationship field kills the concept of modular classes, remember a class should contain related attributes

Reply:
The story field is not part of the user model, its part of the story model if you are to look at the models file, there is User model class and Story model class.